### ISSUE TITLE
filecatalogcli: `dataset` and `help dataset` do not match

### ISSUE THREAD

**Issue by guiguem** [DIRACGrid/DIRAC#3306]
It seems that the help provided when writing `help dataset` and `dataset` in the file catalog CLI do not match.
The first one returns:
```
A set of dataset manipulation commands
    
        Usage:
          
          dataset add <dataset_name> <meta_query>          - add a new dataset definition
          dataset annotate <dataset_name> <annotation>     - add annotation to a dataset
          dataset show [-l] [<dataset_name>]               - show existing datasets
          dataset status <dataset_name>                    - display the dataset status
          dataset files <dataset_name>                     - show dataset files     
          dataset rm <dataset_name>                        - remove dataset
          dataset check <dataset_name>                     - check if the dataset parameters are still valid     
          dataset update <dataset_name>                    - update the dataset parameters
          dataset freeze <dataset_name>                    - fix the current contents of the dataset     
          dataset release <dataset_name>                   - release the dynamic dataset
```
and the second one returns:
```Metadata related operations
    
        Usage:
          meta index [-d|-f|-r] <metaname> [<metatype>]  - add new metadata index. Possible types are:
                                                           'int', 'float', 'string', 'date';
                                                         -d  directory metadata
                                                         -f  file metadata
                                                         -r  remove the specified metadata index
          meta set <path> <metaname> <metavalue> [<metaname> <metavalue> ...]- set metadata values for directory or file
          meta remove <path> <metaname> [<metaname> ...] - remove metadata values for directory or file
          meta get [-e] [<path>] - get metadata for the given directory or file
          meta tags <path> <metaname> where <meta_selection> - get values (tags) of the given metaname compatible with 
                                                        the metadata selection
          meta show - show all defined metadata indice
```
This is due to some mistake in do_dataset in DataManagementSystem/Client/FileCatalogClientCLI.py: `self.do_meta.__doc__` is used instead of `self.do_dataset.__doc__ `

**Comment by guiguem** [comment]
In a very similar way, `do_mkdir` does not have the right doc associated (has `do_group.__doc__`)

**Comment by atsareg** [comment]
Fixed in #3313
